projects/aqlprofile/src/core/memorymanager.hpp

```diff
@@ -196,6 +196,24 @@ class TraceMemoryManager : public MemoryManager {
                      aqlprofile_memory_dealloc_callback_t dealloc, void* data)
       : MemoryManager(agent, alloc, dealloc, data) {}
 
+  void* AddExtraOutputBuf()
+  {
+    aqlprofile_buffer_desc_flags_t flags{};
+    flags.device_access = true;
+    flags.memory_hint = AQLPROFILE_MEMORY_HINT_DEVICE_NONCOHERENT;
+    extra_output_buffers.emplace_back(AllocMemory(outputbuf_size, flags));
+    return extra_output_buffers.back().get();
+  }
+  
+  void* AddExtraCmdBuf(size_t size)
+  {
+    aqlprofile_buffer_desc_flags_t flags{};
+    flags.host_access = flags.device_access = true;
```

Comment:
Nit-picking. Somehow, find it easier to read.
```suggestion
    flags.host_access = true;
    flags.device_access = true;
```

Reply:
done